### ISSUE TITLE
Cursor inside bracket after tab autocomplete

### ISSUE THREAD

**Issue by PixelT** [marijnh/tern_for_sublime#86]
For example:
I have a code:
```
var x = 'example string';
```
now I write the:
```
x.
```
and select one of existing method, using autocomplete by tab (for example I select slice)
after using tab, I get the cursor after the parenthesis ( | is cursor )
```
x.slice()|
```
I should / would like to receive the cursor inside parenthesis (+ automatically open tooltip / popup with hint):
```
x.slice(|)
```

EDIT
Or if I have:
```
"tern_argument_completion": false
```
and select from autocomplete the method (fn), the parenthesis shouldby added automatically

**Comment by marijnh** [comment]
I don't know if we can instruct Sublime Text to put the cursor in a given position after completion. If you know of a way, a patch would be welcome.

The argument completion was added in 3bfe80e993f1 by @MANGO1234. Maybe they can take a look?

**Comment by MANGO1234** [comment]
Using ${1} will place cursor there, tooltip will display automatically, pr here https://github.com/marijnh/tern_for_sublime/pull/89

**Comment by PixelT** [comment]
@MANGO1234 i quickly tested your patch - works brilliant. Thanks!

**Comment by marijnh** [comment]
Solved via #89
